.circleci/config.yml

```diff
@@ -0,0 +1,61 @@
+version: 2
+jobs:
+  build:
+    working_directory: /go/src/k8s.io/helm
+    parallelism: 4
+    docker:
+      - image: golang:1.8
+    environment:
+      PROJECT_NAME: "kubernetes-helm"
+      DOCKER_VERSION: "17.03.0-ce"
```

Comment:
I think the latest version is `17.06.1`, but I am not sure if the docker running on circle is the same version

Reply:
This will only affect client version, doesn't really matter. circle default docker engine set up with `setup_remote_docker` is 17.03.0-ce